Web/src/components/games/MentalSums/NumKeypad.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const NUMPAD_KEYS = ['7', '8', '9', '4', '5', '6', '1', '2', '3', '-', '0'];
+
+const KeypadWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const KeypadContainer = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+`;
+
+const KeyButton = styled.button`
+  background: rgba(255, 255, 255, 0.5);
+  border-radius: 15px;
+  padding: 0.6rem 2rem;
+  margin: 0.3rem;
+  flex: 0 0 calc((100% / 3) - 0.6rem);
+
+  color: var(--dark-purple);
+  font-size: 2rem;
+`;
+
+type NumKeypadProps = {
+  onClickKey: (key: string) => void;
+};
+
+const NumKeypad: React.FC<NumKeypadProps> = ({ onClickKey }) => (
+  <KeypadWrapper>
+    <KeypadContainer>
+      {
+        NUMPAD_KEYS.map((val, index) => (
+          <KeyButton
+            // eslint-disable-next-line react/no-array-index-key
+            key={index}
```

Comment:
`val` should be unique I think, so can use that as a key instead, and remove the `eslint-disable-next-line`
```suggestion
            key={val}
```

Reply:
HAHA I didn't do this because this assumes that `val` is unique (which it is in this case la). But i used index since we won't be sorting/adding/removing items from `NUMPAD_KEYS`? 
But I'll just change it 😆

Reply:
Hahah yeah key needs to be unique and in this case `val` is too. Since we won’t be adding/removing buttons, the key is a bit irrelevant, but just annoying to see the warning in console, and annoying to see `eslint-disable-next-line` in code too hahah

Reply:
HAHAHA ye i agree it's annoying to see `eslint-disable-next-line` so changed it hurhur